doc/release/RELEASE-NOTES.md

```diff
@@ -24,6 +24,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-1165](https://opensupplyhub.atlassian.net/browse/OSDEV-1165) - Updated the release protocol to include information about quick fixes and how to perform them. Additionally, updated the GitFlow diagram to visually depict this process.
 * Updated the `RELEASE-PROTOCOL.md` file to include information about OpenSearch and Logstash, stating that their functionality should also be checked after deployment.
 * [OSDEV-1169](https://opensupplyhub.atlassian.net/browse/OSDEV-1169) - Activated deployment database-anonymizer to production.
+* [OSDEV-1177](https://opensupplyhub.atlassian.net/browse/OSDEV-1177) - Clear OpenSearch and stop Logstash during Postgres DB Restore/Reset in Pre-prod/Test/Dev env.
```

Comment:
1. It should be something like this:
Implemented clearing OpenSearch and stopping Logstash during Postgres DB restore/reset in pre-prod/test/dev environments to freshly populate OpenSearch with data from the restored or reset Postgres DB.
You need to show what you have implemented and explain its purpose.
2. This should be moved to version 1.20. Could you please create a new section for version 1.20 using [this template](https://github.com/opensupplyhub/open-supply-hub/blob/main/doc/release/RELEASE-NOTES-TEMPLATE.md)?

Reply:
done